.github/workflows/cypress-test.yml

```diff
@@ -70,3 +70,5 @@ jobs:
           cd opensearch-dashboards-functional-test
           npm install cypress --save-dev
           yarn cypress:run-with-security-and-aggregation-view --browser chrome --spec "cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js"
+          yarn cypress:run-with-security --browser chrome --spec "cypress/integration/plugins/security-dashboards-plugin/multi_tenancy.js"
```

Comment:
Since the directory of these tests are the same, I think you can just replace the original command line with`yarn cypress:run-with-security --browser chrome --spec "cypress/integration/plugins/security-dashboards-plugin/*.js"`

Reply:
Oh nvm, I think some of the flag are not the same, cuz your cases are `run-with-security`. So lets just leave like this.